admiral/pkg/clusters/handler.go

```diff
@@ -323,3 +325,94 @@ func GetServiceWithSuffixMatch(suffix string, services []*coreV1.Service) string
 	}
 	return ""
 }
+
+func HandleEventForClientDiscovery(ctx context.Context, event admiral.EventType, obj *common.K8sObject,
```

Comment:
can this func be moved to `clientdiscovery_handler.go`?

Reply:
Done